### PR TITLE
cli: rename spawn -> set-outputs

### PR DESCRIPTION
**Pull request by oliver-sanders**
Catch a few spawn references missed in https://github.com/cylc/cylc-flow/pull/3816.

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (why?).
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.
- [x] No dependency changes.

### CODE REVIEW

**Review by kinow — APPROVED** [review]

**Review by dwsutherland — APPROVED** [review]
Works for me 👍   .. I'll rebase https://github.com/cylc/cylc-flow/pull/3757 after merging this..

File: cylc/flow/scripts/cylc_help.py [hunk 1]
```diff
@@ -238,7 +238,7 @@ def category_help(category):
 control_commands['kill'] = ['kill']
 control_commands['hold'] = ['hold']
 control_commands['release'] = ['release', 'unhold']
-control_commands['spawn'] = ['spawn']
+control_commands['set-outputs'] = ['set-outpus']
```
Comment (hjoliver) [inline cylc/flow/scripts/cylc_help.py:241]: Doh, spelling error!  I'll push up a hot fix, not worth a PR.

Comment (hjoliver) [reply cylc/flow/scripts/cylc_help.py:241]: (Found post merge).

Comment (dwsutherland) [reply cylc/flow/scripts/cylc_help.py:241]: Opps

Comment (oliver-sanders) [reply cylc/flow/scripts/cylc_help.py:241]: 🍌

**Pull request merged**